### PR TITLE
Add changes sub group field to store Estimate or Actual value

### PR DESCRIPTION
**Pull request by AnnaGolosova**
## Link to JIRA ticket

https://app.clickup.com/t/25cf7m4

### *What is the problem we're trying to solve*

As a system I want to know is the carge related to Estimate or Actual group

### *What changes have we introduced*

Modify API to add charge_sub_group field

#### _Checklist_

- [x] Added tests to cover all new production code
- [x] Checked all code for possible refactoring
- [x] Code pipeline builds correctly

### CODE REVIEW

File: ChargesApi/V1/Boundary/Request/AddChargeRequest.cs [hunk 1]
```diff
@@ -8,15 +8,25 @@ namespace ChargesApi.V1.Boundary.Request
...
         public TargetType TargetType { get; set; }
 
+        /// <example>Leaseholders</example>
         [AllowedValues(typeof(ChargeGroup))]
         public ChargeGroup ChargeGroup { get; set; }
 
+        /// <summary>
+        /// Allowed values - Estimate/Actual
+        /// </summary>
+        /// <example>Actual</example>
```
Comment (sandipchandra) [inline ChargesApi/V1/Boundary/Request/AddChargeRequest.cs:26]: Put allowed values attribute please

**Review by sandipchandra — COMMENTED** [review]
Please review the comments

File: ChargesApi/V1/Boundary/Request/AddChargeRequest.cs (hunk 1 above)
Comment (AnnaGolosova) [reply ChargesApi/V1/Boundary/Request/AddChargeRequest.cs:27]: I've added Vluent Validation to check that Charge Sub Group should be provided if Charge Group is Tenants. Can you please review again?

**Review by sandipchandra — APPROVED** [review]
approved

**Pull request merged**